### ISSUE TITLE
Crash: NSInternalInconsistencyException in ReaderDetailViewController.viewWillTransition

### ISSUE THREAD

**Issue by SiobhyB** [wordpress-mobile/WordPress-iOS#10571]
```
Fatal Exception: NSInternalInconsistencyException
0  CoreFoundation                 0x212cebea0 __exceptionPreprocess
1  libobjc.A.dylib                0x211ebda40 objc_exception_throw
2  CoreFoundation                 0x212c01c1c +[_CFXNotificationTokenRegistration keyCallbacks]
3  Foundation                     0x2136ef140 -[NSAssertionHandler handleFailureInMethod:object:file:lineNumber:description:]
4  Foundation                     0x213689cd4 -[NSLayoutConstraint _setSymbolicConstant:constant:]
5  Foundation                     0x2136899f4 -[NSLayoutConstraint setConstant:]
6  UIKitCore                      0x240315760 -[UIScrollView setContentOffset:]
7  UIKitCore                      0x240216a98 -[UITextView setContentOffset:]
8  UIKitCore                      0x24032d93c -[UIScrollView(UIScrollViewInternal) _setContentOffset:animated:animationCurve:animationAdjustsForContentOffsetDelta:animation:]
9  WordPress                      0x102dd3274 closure #1 in ReaderDetailViewController.viewWillTransition(to:with:) (<compiler-generated>)
10 WordPress                      0x102ff2c30 thunk for @escaping @callee_guaranteed (@guaranteed UIViewControllerTransitionCoordinatorContext) -> () (<compiler-generated>)
11 UIKitCore                      0x23f954224 -[_UIViewControllerTransitionCoordinator _applyBlocks:releaseBlocks:]
12 UIKitCore                      0x23f95061c -[_UIViewControllerTransitionContext __runAlongsideAnimations]
13 UIKitCore                      0x2403868b4 +[UIView(UIViewAnimationWithBlocks) _setupAnimationWithDuration:delay:view:options:factory:animations:start:animationStateGenerator:completion:]
14 UIKitCore                      0x240386df8 +[UIView(UIViewAnimationWithBlocks) animateWithDuration:delay:options:animations:completion:]
15 UIKitCore                      0x23f9681b4 __58-[_UIWindowRotationAnimationController animateTransition:]_block_invoke_2
16 UIKitCore                      0x24038b278 +[UIView(Internal) _performBlockDelayingTriggeringResponderEvents:]
17 UIKitCore                      0x23f967f40 __58-[_UIWindowRotationAnimationController animateTransition:]_block_invoke
18 UIKitCore                      0x2403868b4 +[UIView(UIViewAnimationWithBlocks) _setupAnimationWithDuration:delay:view:options:factory:animations:start:animationStateGenerator:completion:]
19 UIKitCore                      0x240386df8 +[UIView(UIViewAnimationWithBlocks) animateWithDuration:delay:options:animations:completion:]
20 UIKitCore                      0x23f967e14 -[_UIWindowRotationAnimationController animateTransition:]
21 UIKitCore                      0x23ff2c328 -[UIWindow _rotateToBounds:withAnimator:transitionContext:]
22 UIKitCore                      0x23ff2ebd8 -[UIWindow _rotateWindowToOrientation:updateStatusBar:duration:skipCallbacks:]
23 UIKitCore                      0x23ff2f3d4 -[UIWindow _setRotatableClient:toOrientation:updateStatusBar:duration:force:isRotating:]
24 UIKitCore                      0x23ff2e67c -[UIWindow _setRotatableViewOrientation:updateStatusBar:duration:force:]
25 UIKitCore                      0x23ff2d2e0 __57-[UIWindow _updateToInterfaceOrientation:duration:force:]_block_invoke
26 UIKitCore                      0x23ff2d17c -[UIWindow _updateToInterfaceOrientation:duration:force:]
27 CoreFoundation                 0x212c5a928 __CFNOTIFICATIONCENTER_IS_CALLING_OUT_TO_AN_OBSERVER__
28 CoreFoundation                 0x212c5a8f4 ___CFXRegistrationPost_block_invoke
29 CoreFoundation                 0x212c59de4 _CFXRegistrationPost
30 CoreFoundation                 0x212c59a90 ___CFXNotificationPost_block_invoke
31 CoreFoundation                 0x212bd1d70 -[_CFXNotificationRegistrar find:object:observer:enumerator:]
32 CoreFoundation                 0x212c5951c _CFXNotificationPost
33 Foundation                     0x213669c10 -[NSNotificationCenter postNotificationName:object:userInfo:]
34 UIKitCore                      0x23fb72bb8 -[UIDevice setOrientation:animated:]
35 UIKitCore                      0x23fef8cd0 _runAfterCACommitDeferredBlocks
36 UIKitCore                      0x23fee6ccc _cleanUpAfterCAFlushAndRunDeferredBlocks
37 UIKitCore                      0x23ff076a4 __34-[UIApplication _firstCommitBlock]_block_invoke_2
38 CoreFoundation                 0x212c7c01c __CFRUNLOOP_IS_CALLING_OUT_TO_A_BLOCK__
39 CoreFoundation                 0x212c7b91c __CFRunLoopDoBlocks
40 CoreFoundation                 0x212c76910 __CFRunLoopRun
41 CoreFoundation                 0x212c761cc CFRunLoopRunSpecific
42 GraphicsServices               0x214eed584 GSEventRunModal
43 UIKitCore                      0x23feed054 UIApplicationMain
44 WordPress                      0x102c2f5ac main (main.m:11)
45 libdyld.dylib                  0x212736bb4 start
```
This was reported to us by @madeincosmos who noted the app crashing when she attempted to open it. It continued to crash when she closed then attempted to reopen the app. The only workaround was for her to delete then reinstall the app.

Maria noted that she mostly uses the app to check notifications and view posts in the Reader. She hasn't tried to write or edit a post on her phone in a very long time.

5be53aabf8b88c29632ad70c-fabric

**Comment by rachelmcr** [comment]
Optimistically closing this as resolved by #10578, since we haven't seen this crash in 11.4. We can reopen this issue if the crash reappears.

**Comment by aerych** [comment]
Fingers crossed! 🤞
